server/middlewares/index.js

```diff
@@ -0,0 +1,26 @@
+function notFound(req, res, next) {
+  const error = new Error(`Not Found - ${req.originalUrl}`);
+  res.status(404);
+  next(error);
+}
+
+// eslint-disable-next-line no-unused-vars
+function errorHandler(error, req, res, next) {
+  const { message, stack } = error;
+  const status = res.statusCode === 200 ? 500 : res.statusCode;
+  res.status(status).json({
+    status: {
+			code: status,
+			type: "error",
+		},
+		error: {
+			message,
+		},
+    stack: process.env.NODE_ENV === 'production' ? '🥞' : stack,
+  });
+}
+
+module.exports = {
+  errorHandler,
+  notFound,
+};
```

Comment:
What is the use-case of this whole middleware?

Reply:
When we send errors via `next` method, those will be handled by these middledwares.

Reply:
But where are you using it exactly?

Reply:
Here: https://github.com/logchimp/logchimp/pull/12/files#diff-9e4a215b4f5f613a802fa43d8ab35270R33

Reply:
I still don't get it, I guess that's fine.

Can you format the middleware according to this? https://github.com/logchimp/logchimp/pull/13/files